src/haz3lweb/view/dhcode/DHCode.re

```diff
@@ -59,6 +59,10 @@ let view_of_layout =
                  [with_cls("InvalidIntOfString", txt)],
                  ds,
                )
+             | OperationError(InvalidFloatOfString) => (
+                 [with_cls("InvalidFloatOfString", txt)],
```

Comment:
let's just add an OperationError class rather than a separate class for each kind of operation error -- we will want to style all errors the same